app/javascript/components/workers-form/helpers.jsx

```diff
@@ -0,0 +1,237 @@
+/**
+ * Converts any value to bytes values.
+ *
+ * @param {any} value Input value.
+ * @return {number} Value in bytes.
+ */
+export const toBytes = (value) => {
+  if (!value) return undefined;
+
+  if (typeof value !== 'number') {
+    const sizeValue = value.match(/^\d+(\.\d+)*/)[0];
+    if (value.match(/gigabytes/)) {
+      return sizeValue * (1024 ** 3);
+    }
+    if (value.match(/megabytes/)) {
+      return sizeValue * (1024 ** 2);
+    }
+    if (value.match(/kilobytes/)) {
+      return sizeValue * 1024;
+    }
+    if (value.match(/.bytes/)) {
```

Comment:
This line has `.bytes`, but the others don't have the leading dot.  Is that intentional?  Also, a dot in Regex means "any character"...what that also intentional or did you mean `\.`?

Reply:
You are right! It should be \.